src/test/java/com/example/universe/simulator/entityservice/unit/controllers/PlanetControllerTest.java

```diff
@@ -0,0 +1,342 @@
+package com.example.universe.simulator.entityservice.unit.controllers;
+
+import com.example.universe.simulator.entityservice.controllers.PlanetController;
+import com.example.universe.simulator.entityservice.dtos.PlanetDto;
+import com.example.universe.simulator.entityservice.entities.Planet;
+import com.example.universe.simulator.entityservice.exception.AppException;
+import com.example.universe.simulator.entityservice.exception.ErrorCodeType;
+import com.example.universe.simulator.entityservice.services.PlanetService;
+import com.example.universe.simulator.entityservice.unit.AbstractWebMvcTest;
+import com.example.universe.simulator.entityservice.utils.TestUtils;
+import org.junit.jupiter.api.Test;
+import org.modelmapper.TypeToken;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.never;
+
+@WebMvcTest(PlanetController.class)
+class PlanetControllerTest extends AbstractWebMvcTest {
```

Comment:
Should include null star and null star id validation tests for add/update operations. Check #63